packages/cli/src/commands/compare/index.ts

```diff
@@ -28,28 +28,31 @@ import {
   headless,
   config,
   report,
-} from '../helpers/flags';
+} from '../../helpers/flags';
 import {
   fidelityLookup,
   headlessFlags,
   defaultFlagArgs,
-} from '../command-config/default-flag-args';
-import { logCompareResults } from '../helpers/log-compare-results';
+} from '../../command-config/default-flag-args';
 import {
   chalkScheme,
   checkEnvironmentSpecificOverride,
   parseMarkers,
-} from '../helpers/utils';
-import { getEmulateDeviceSettingForKeyAndOrientation } from '../helpers/simulate-device-options';
+} from '../../helpers/utils';
+import { getEmulateDeviceSettingForKeyAndOrientation } from '../../helpers/simulate-device-options';
 import {
   CONTROL_ENV_OVERRIDE_ATTR,
   EXPERIMENT_ENV_OVERRIDE_ATTR,
   ITBConfig,
-} from '../command-config/tb-config';
-import Report from './report';
+} from '../../command-config/tb-config';
 import { IConfig } from '@oclif/config';
-import { getConfig } from '../command-config/build-config';
+import { getConfig } from '../../command-config';
+import CompareAnalyze from './analyze';
+import Report from '../report';
+import chalk from 'chalk';
```

Comment:
Instead of importing chalk here we can import the `chalkScheme` from https://github.com/TracerBench/tracerbench/blob/master/packages/cli/src/helpers/utils.ts#L214-L230